runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -1069,13 +1036,20 @@ public void onItemContainerChanged(ItemContainerChanged itemContainerChanged)
 		ItemContainer container = itemContainerChanged.getItemContainer();
 
 		Item weapon = container.getItem(EquipmentInventorySlot.WEAPON.getSlotIdx());
-		if (weapon == null ||
-			(weapon.getId() != ItemID.STAFF_OF_THE_DEAD &&
-				weapon.getId() != ItemID.TOXIC_STAFF_OF_THE_DEAD &&
-				weapon.getId() != ItemID.STAFF_OF_LIGHT &&
-				weapon.getId() != ItemID.TOXIC_STAFF_UNCHARGED))
+		if (weapon != null &&
+				(weapon.getId() == ItemID.STAFF_OF_THE_DEAD ||
+						weapon.getId() == ItemID.TOXIC_STAFF_OF_THE_DEAD ||
+						weapon.getId() == ItemID.STAFF_OF_LIGHT ||
+						weapon.getId() == ItemID.STAFF_OF_BALANCE ||
+						weapon.getId() == ItemID.TOXIC_STAFF_UNCHARGED) &&
+				staffProtectionExpireTick > client.getTickCount())
+		{
+			// Recreate timer with remaining duration if a protective staff is re-equipped within 1 minute since activation
```

Comment:
This code existed because our previous understanding of the special effect was wrong--we believed that unequipping the staff caused the effect to end rather than to stop working only when it was unequipped. Instead of removing/recreating the timer, just leave it present at all times regardless of whether a sotd variant is equipped.

Reply:
We discussed this on Discord and decided we'd like the infobox to remain visible even if a staff is unequipped, though it'd be nice to make the box's image different when a staff is not equipped--partially transparent perhaps. Currently, the infobox API doesn't offer a way to change the image--to achieve that effect, it would need to be removed and added again with a changed image. We are also trying to move away from code which removes and re-adds infoboxes since that causes reordering of other active infoboxes, so some API changes are needed to make this happen the way we'd like.